Protos/build_protos.sh

```diff
@@ -15,24 +15,22 @@
 # limitations under the License.
 
 
-
 #This is a convenience script to build protos and generate Swift files
```

Comment:
A pointer to a step-by-step process that would work for anyone would be better here - but not a high priority

Reply:
I've updated the Readme file with instructions for generating the Swift files.